eks_fargate/tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+minversion = 2.0
+skip_missing_interpreters = true
+basepython = py37
+envlist =
+    py{27,37}-eks_fargate
```

Comment:
```suggestion
    py{27,37}
```

Reply:
What's that for ?

Reply:
Just consistency with what we mostly have in the rest of the repo